docs/registries.md

```diff
@@ -103,27 +103,30 @@ If you don't want k3d to manage your registry, you can start it with some `docke
 
 ```shell script
 docker volume create local_registry
-docker container run -d --name registry.local -v local_registry:/var/lib/registry --restart always -p 5000:5000 registry:2
+docker container run -d --name registry.localhost -v local_registry:/var/lib/registry --restart always -p 5000:5000 registry:2
 ```
 
-These commands will start you registry in `registry.local:5000`. In order to push to this registry, you will
+These commands will start you registry in `registry.localhost:5000`. In order to push to this registry, you will
 need to add the line at `/etc/hosts` as we described in [the previous section ](#etc-hosts). Once your
 registry is up and running, we will need to add it to your [`registries.yaml` configuration file](#registries-file).
 Finally, you must connect the registry network to the k3d cluster network:
-`docker network connect k3d-k3s-default registry.local`. And then you can
+`docker network connect k3d-k3s-default registry.localhost`. And then you can
 [check you local registry](#testing).
 
 ### <a name="etc-hosts"></a>Pushing to your local registry address
 
-The registry will be located, by default, at `registry.local:5000` (customizable with the `--registry-name`
+The registry will be located, by default, at `registry.localhost:5000` (customizable with the `--registry-name`
 and `--registry-port` parameters). All the nodes in your k3d cluster can resolve this hostname (thanks to the
 DNS server provided by the Docker daemon) but, in order to be able to push to this registry, this hostname
 but also be resolved from your host.
 
-The easiest solution for this is to add an entry in your `/etc/hosts` file like this:
+Luckily, (NSS-myhostname)[http://man7.org/linux/man-pages/man8/nss-myhostname.8.html] should be installed on a desktop environment
+and should resolve `*.localhost` automatically to `127.0.0.1`.
+
+If it's not the case, you can add an entry in your `/etc/hosts` file like this:
```

Comment:
Maybe add a note here, how you can install it (`sudo apt install libnss-myhostname`)

Reply:
done